tests/playbooks/tests_wireless_plugin_installation.yml

```diff
@@ -3,7 +3,11 @@
 - name: Play for testing wireless plugin installation
   hosts: all
   tasks:
-    - name: Check if rpm ostree system - cannot test
+    - name: >-
+        This is an rpm-ostree system.  This test cannot be run on such a system.
+        The test wants to test removing and reinstalling the
+        NetworkManager-wifi package.  Package removal is not allowed on
+        rpm-ostree systems.
```

Comment:
maybe we can include the task file here with `var: {{package}}`, then in the task file itself, we can write it as :

```
    - name: Cannot test ostree systems because the package can not be removed
        or installed - '{{ package}}'
      meta: end_host
      when: __network_is_ostree | d(false) 
  ```

Reply:
> maybe we can include the task file here

which task file?  I'm not sure what you mean.

> with `var: {{package}}`, then in the task file itself, we can write it as :
> 
> ```
>     - name: Cannot test ostree systems because the package can not be removed
>         or installed - '{{ package}}'
>       meta: end_host
>       when: __network_is_ostree | d(false) 
> ```

Reply:
Put the following task into a separate task file, for example, `tasks/ostree_systems_check.yml`
```
    - name: Cannot test ostree systems because the package can not be removed
        or installed - '{{ package}}'
      meta: end_host
      when: __network_is_ostree | d(false) 
```

Then in the playbook (tests/playbooks/*.yml), we can include the task file as: 
```
    - name: Include the task 'ostree_systems_check.yml'
      include_tasks: tasks/ostree_systems_check.yml
```

Reply:
If using the task file `tasks/ostree_systems_check.yml`, we can also include the task file in  `tests_wireless_and_network_restart.yml`.